modules/eslint-plugin/package.json

```
@@ -44,6 +44,6 @@
   },
   "peerDependencies": {
     "eslint": ">=8.0.0",
-    "typescript": ">=4.4.0"
+    "typescript": ">=4.4.0 || >=5.0.0"
```

Let's use a `*` so we don't bump into this anymore.
It seems like the angular eslint project is also using this https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin/package.json#L25C1-L25C22

```suggestion
    "typescript": "*"
```

---

Praise: Good solution, no need to worry about it again